src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -153,6 +153,37 @@ def test_batch_application_cmd(self, resource_group):
         self.cmd('storage account delete -g {rg} -n {str_n} --yes')
 
 
+class BatchMgmtNetworkScenarioTests(ScenarioTest, BatchScenarioMixin):
+    def _get_test_data_file(self, filename):
+        filepath = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', filename)
+        self.assertTrue(os.path.isfile(filepath), 'File {} does not exist.'.format(filepath))
+        return filepath
+
+    @ResourceGroupPreparer(location='westeurope')
+    def test_batch_network_cmd(self, resource_group):
+        self.kwargs.update({
+            'rg': resource_group,
+            'loc': 'westeurope',
+            'acc': 'clibatchtest9',
+            'pool_file': self._get_test_data_file('batch-pool-with-network-config.json')
+        })
+
+        # Create account
+        self.cmd('batch account create -g {rg} -n {acc} -l {loc}').assert_with_checks([
+            self.check('name', '{acc}'),
+            self.check('location', '{loc}'),
+            self.check('resourceGroup', '{rg}')])
+        self.set_account_info('clibatchtest9', resource_group)
+        self.cmd('batch account login -g {rg} -n {acc}')
+
+        # Get operations on private-link-resource
+        self.cmd('batch private-link-resource list -g {rg} --account-name {acc}')
```

Comment:
Looks like the service is not ready yet. What's the plan for the feature and this PR?

Reply:
The API is GA, but private link/private endpoint is currently in a whitelisted accounts only mode so essentially preview.